PMM-7-mcp-server (FB)

BupycHuk opened the pull request:
Custom branches: 
pmm

BupycHuk committed:
Create feature build: PMM-7-mcp-server

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-397d15e
Client docker: perconalab/pmm-client-fb:PR-3934-397d15e
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-397d15e
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-397d15e.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-397d15e&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-397d15e.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/2753/

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-500a697
Client docker: perconalab/pmm-client-fb:PR-3934-500a697
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-500a697
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-500a697.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-500a697&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-500a697.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/2778/

BupycHuk committed:
Merge branch 'v3' into PMM-7-mcp-server

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-017c69a
Client docker: perconalab/pmm-client-fb:PR-3934-017c69a
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-017c69a
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-017c69a.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-017c69a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-017c69a.tar.gz

JNKPercona commented:
API tests have succeded: https://pmm.cd.percona.com/job/pmm3-api-tests/2850/

BupycHuk committed:
Merge branch 'v3' into PMM-7-mcp-server

BupycHuk committed:
Merge branch 'v3' into PMM-7-mcp-server

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
Merge branch 'v3' into PMM-7-mcp-server

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
Add 'git pull' step to the 'trigger' target in Makefile

Ensures the local repository is up-to-date before creating an empty commit to trigger the build. This prevents potential issues caused by outdated local branches during the process.

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-dc31ffd
Client docker: perconalab/pmm-client-fb:PR-3934-dc31ffd
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-dc31ffd
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-dc31ffd.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-dc31ffd&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-dc31ffd.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3037/

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-4b8d886
Client docker: perconalab/pmm-client-fb:PR-3934-4b8d886
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-4b8d886
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-4b8d886.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-4b8d886&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-4b8d886.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3066/

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-6aa784b
Client docker: perconalab/pmm-client-fb:PR-3934-6aa784b
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-6aa784b
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-6aa784b.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-6aa784b&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-6aa784b.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3067/

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
Merge branch 'v3' into PMM-7-mcp-server

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-5c2b49a
Client docker: perconalab/pmm-client-fb:PR-3934-5c2b49a
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-5c2b49a
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-5c2b49a.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-5c2b49a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-5c2b49a.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3073/

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-48393c9
Client docker: perconalab/pmm-client-fb:PR-3934-48393c9
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-48393c9
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-48393c9.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-48393c9&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-48393c9.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3087/

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
Add Grafana configuration to CI pipeline

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-b24f241
Client docker: perconalab/pmm-client-fb:PR-3934-b24f241
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-b24f241
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-b24f241.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-b24f241&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-b24f241.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3170/

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-4c8bca6
Client docker: perconalab/pmm-client-fb:PR-3934-4c8bca6
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-4c8bca6
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-4c8bca6.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-4c8bca6&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-4c8bca6.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3174/

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-55af84d
Client docker: perconalab/pmm-client-fb:PR-3934-55af84d
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-55af84d
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-55af84d.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-55af84d&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-55af84d.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3179/

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-4380fe8
Client docker: perconalab/pmm-client-fb:PR-3934-4380fe8
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-4380fe8
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-4380fe8.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-4380fe8&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-4380fe8.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3180/

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-2fd4ad4
Client docker: perconalab/pmm-client-fb:PR-3934-2fd4ad4
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-2fd4ad4
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-2fd4ad4.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-2fd4ad4&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-2fd4ad4.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3193/

BupycHuk committed:
chore: trigger FB

BupycHuk committed:
chore: trigger FB

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3934-8a0c66d
Client docker: perconalab/pmm-client-fb:PR-3934-8a0c66d
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3934-8a0c66d
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-8a0c66d.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3934-8a0c66d&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3934-8a0c66d.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/3202/
